cli/src/main/java/dev/starfix/Starfix.java

```diff
@@ -134,6 +152,44 @@ public static File getConfigFile() {
         return new File(userHome + "/.config/starfix.yaml");
 
     }
+    // Function to load default config 
+    public void defaultConfig() {
+        String path_env = System.getenv("Path"); // System PATH variable
+        clone_path =  System.getProperty("user.home") + "/code"; // set clone_path to /home/user_name/code
+        File configFile = getConfigFile();
+
+        try {
+            configFile.createNewFile();
+            
+            if(isWindows()){// check if Windows OS
+                if(path_env.contains("Microsoft VS Code")){ // If PATH has VScode
+                    ide = "code.cmd";
+                    writeToYAMLFile(ide, clone_path,configFile);
+                } else if(path_env.contains("IntelliJ IDEA")){ // If PATH has IntelliJ
+                    ide = "idea64.exe";
+                    writeToYAMLFile(ide, clone_path,configFile);
+                }
```

Comment:
We shouldn't have to write the default config to disk. Then it by definition is no longer a default.